specification/containerservices/resource-manager/readme.md

```diff
@@ -221,12 +219,40 @@ Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azur
 csharp:
   azure-arm: true
   license-header: MICROSOFT_MIT_NO_VERSION
-  namespace: Microsoft.Azure.Management.ContainerServices
+  namespace: Microsoft.Azure.Management.ContainerService
   payload-flattening-threshold: 1
-  output-folder: $(csharp-sdks-folder)/ContainerServices/Management.ContainerServices/Generated
+  output-folder: $(csharp-sdks-folder)/ContainerService/Management.ContainerService/Generated
   clear-output-folder: true
 ```
 
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml $(python)
+python-mode: create
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: azure.mgmt.containerservice
+  package-name: azure-mgmt-containerservice
+  package-version: 1.0.0
+  clear-output-folder: true
+```
+``` yaml $(python) && $(python-mode) == 'update'
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/azure-mgmt-containerservice/azure/mgmt/containerservice
```

Comment:
Any reason to name the package containerServices (plural) in C# and containerservice (singular) in pythion?

Reply:
They should both be singular as that is reflective of the ARM resource name.